### PR TITLE
MaNGA data + stellar abundance plots

### PR DESCRIPTION
**Pull request by correac**
I am adding the new MaNGA dataset to the stellar mass-metallicity relation. I have also added stellar abundance ratio plots, [Si/Fe], [Ne/Fe], [Sr/Fe], [Ba/Fe] and [Eu/Fe] vs [Fe/H].

### CODE REVIEW

File: colibre/auto_plotter/metallicity.yml [hunk 1]
```diff
@@ -507,17 +507,18 @@ stellar_mass_star_metallicity_50:
...
     title: "Stellar mass - Star metallicity relation (50 kpc aperture)"
-    caption: Average stellar metallicity measured in the same aperture as the stellar mass. Gallazzi data is corrected from their choice of solar metallicity (0.02) to ours (0.0126).
+    caption: Average stellar metallicity measured in the same aperture as the stellar mass. Gallazzi data is corrected from their choice of solar metallicity (0.02) to ours (0.0126). The Yates+ data from MaNGA corresponds to mass-weighted stellar metallicity within 3 arcsec for star-forming galaxies at 𝑧 = 0.
     section: Stellar Metallicity
     show_on_webpage: true
   observational_data:
     - filename: GalaxyStellarMassStellarMetallicity/Gallazzi2005_Data.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Kirby2013_Data.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Fraser-McKelvie_2021.hdf5
-    - filename: GalaxyStellarMassStellarMetallicity/FIREbox.hdf5
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/metallicity.yml:517]: If you'd like to remove FIREbox, can we please discuss it separately and then possibly add these changes as part of another pull request?

The current pull request is supposed to do only one thing: add the new data, not remove the existing one.

File: colibre/auto_plotter/metallicity.yml [hunk 2]
```diff
@@ -555,10 +556,11 @@ stellar_mass_star_metallicity_30:
...
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/metallicity.yml:558]: Same comment regarding the FIREbox data as above.

File: colibre/auto_plotter/metallicity.yml [hunk 3]
```diff
@@ -507,17 +507,18 @@ stellar_mass_star_metallicity_50:
       units: "dimensionless"
   metadata:
     title: "Stellar mass - Star metallicity relation (50 kpc aperture)"
-    caption: Average stellar metallicity measured in the same aperture as the stellar mass. Gallazzi data is corrected from their choice of solar metallicity (0.02) to ours (0.0126).
+    caption: Average stellar metallicity measured in the same aperture as the stellar mass. Gallazzi data is corrected from their choice of solar metallicity (0.02) to ours (0.0126). The Yates+ data from MaNGA corresponds to mass-weighted stellar metallicity within 3 arcsec for star-forming galaxies at 𝑧 = 0.
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/metallicity.yml:510]:
```suggestion
    caption: Average stellar metallicity measured in the same aperture as the stellar mass. Gallazzi data is corrected from their choice of solar metallicity (0.02) to ours (0.0134). The Yates+ data from MaNGA corresponds to mass-weighted stellar metallicity within 3 arcsec for star-forming galaxies at $z=0$.
```

**Review by EvgeniiChaikin — CHANGES_REQUESTED** [review]
Hi @correac thank you for the update!

And sorry that having a look at your PR took a bit more time than usual!

I added four small comments to this pull request.

Note also that in this PR, you need to update the version of the observational data repository in order to include the MANGA data.

The other thing is that the script `stellar_abundances.py` has now grown very large. Hence, It'd be great to optimize the code in it. I will try to soon have a closer look and shorten the code wherever possible; I will then submit a PR with the changes and select you as a reviewer.

File: colibre/auto_plotter/metallicity.yml (hunk 3 above)
Comment (correac) [reply colibre/auto_plotter/metallicity.yml]: Added!

File: colibre/auto_plotter/metallicity.yml (hunk 2 above)
Comment (correac) [reply colibre/auto_plotter/metallicity.yml]: Oh sorry, I took it out and forgot to put it back in.

**Comment by EvgeniiChaikin** [comment]
Thank you @correac, all code looks good!

No, `6698966` is not the latest version (see the screenshot below)

Screenshot from 2023-01-11 12-12-36

**Comment by correac** [comment]
Hi @EvgeniiChaikin oh you are right! I was looking at the wrong repository. Now I updated it for obs. data to be checked out '2e291bc9011...'

**Review by EvgeniiChaikin — APPROVED** [review]

**Comment by MatthieuSchaller** [comment]
Thanks!

**Pull request merged**